src/main/scala/cromwell/engine/backend/jes/GcsAuth.scala

```diff
@@ -0,0 +1,71 @@
+package cromwell.engine.backend.jes
+
+import play.api.libs.json._
```

Comment:
Why did you choose to use this JSON library as opposed to Spray JSON which we use in the rest of our code?